script/funcenv

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -eux
+
+#
+# Script to run tests
+#
+
+top_srcdir=$(readlink -m $0/../..)
+cd ${top_srcdir}
+
+export PITRERY_BACKUP_DIR=/tmp/backup
+
+teardown() {
+	exit_code=$?
+
+	# If not on CI and we are docker entrypoint (PID 1), let's wait forever on
+	# error. This allows user to enter the container and debug after a build
+	# failure.
+	if [ -z "${CI-}" -a $PPID = 1 -a $exit_code -gt 0 ] ; then
+		tail -f /dev/null
+	fi
+
+	# stop and destroy PG cluster
+	sudo -su postgres ${TBD_PGBIN}/pg_ctl stop -w -D /tmp/pgsql
+	rm -rf /tmp/pgsql
+
+	rm -rf $PITRERY_BACKUP_DIR
+
+	exit ${exit_code}
+}
+
+trap teardown EXIT INT TERM
+
+export TBD_PGBIN=$(readlink -e /usr/pgsql-${TBD_PGVERSION}/bin /usr/lib/postgresql/${TBD_PGVERSION}/bin)
```

Comment:
TBD = TemBoarD. Je suggère de retirer `TBD_` carrément.

Reply:
Aaah, ok, je pensais que c'était "To Be Defined"  :-p